Makefile

```diff
@@ -673,19 +674,33 @@ doc-html doc-latexpdf doc-epub doc-livehtml : doc-% : doxygen
 
 doc : doc-html
 
-style-c :
-	@astyle --options=.astylerc \
-          $(filter-out include/ceedf.h $(wildcard tests/t*-f.h), \
-            $(wildcard include/*.h interface/*.[ch] tests/*.[ch] backends/*/*.[ch] \
-              examples/*/*/*.[ch] examples/*/*.[ch] examples/*/*.[ch]pp gallery/*/*.[ch]))
+# Style/Format
+CLANG_FORMAT ?= clang-format
+
+FORMAT_OPTS += -style=file -i
+
+%.format : %
+	$(CLANG_FORMAT) $(FORMAT_OPTS) $^
+
+format.ch := $(filter-out include/ceedf.h $(wildcard tests/t*-f.h), \
+            $(wildcard include/*.h include/*/*.h include/*/*/*.h include/*/*/*/*.h \
+              interface/*.[ch] tests/*.[ch] backends/*/*.[ch] backends/*/*/*.[ch] backends/*/*.[ch]pp backends/*/*/*.[ch]pp \
+              examples/*/*/*.[ch] examples/*/*.[ch] examples/*/*.[ch]pp gallery/*.[ch] gallery/*/*.[ch]))
+
+format-c  :  $(format.ch:%=%.format)
+
+style-c   : format-c
```

Comment:
Let's pick one name (`format` or `style`). I don't think the backward compatibility is worth having multiple aliases at this stage.

Reply:
I have zero preferences. Opinions? Ratel has both. Not sure what PETSc uses, but that could be the tiebreaker

Reply:
PETSc `make clangformat`
MFEM `make style` (uses astyle)
Apache Arrow `make format` (uses clang-format)

It doesn't matter to me because I'll very rarely run it manually (that's my editor's job).